libs/datasets/timeseries.py

```diff
@@ -682,7 +682,9 @@ def to_csv(self, path: pathlib.Path):
             provenance_path = str(path).replace(".csv", "-provenance.csv")
             self.provenance.sort_index().rename(PdFields.PROVENANCE).to_csv(provenance_path)
 
-    def join_columns(self, other: "MultiRegionTimeseriesDataset") -> "MultiRegionTimeseriesDataset":
+    def join_columns(
+        self, other: "MultiRegionTimeseriesDataset", replace: bool = False
+    ) -> "MultiRegionTimeseriesDataset":
```

Comment:
We now have test_positivity in the input combined dataset (as read from CMS) but then join in the test_positivity output from test_positivity.py, so I changed this to allow replacing the existing column (instead of erroring).

Alternatively we could have two different test_positivity columns, but I'm not sure if that's better or what to name them (test_positivity_in and test_positivity_out, or something?)

Reply:
Having two test_positivity names seems pretty confusing. Another alternative is having test_positivity.py read the dataset instead of putting TEST_POSITIVITY in combined datasets.
How about `ts.drop_column_if_present(CommonFields.TEST_POSITIVITY).join_columns(test_positivity_results.test_positivity)`? It seems odd that you want to get test positivity for all regions from combined datasets or test_positivity_results instead of blending per region.

Reply:
Thanks!  I'm down with `drop_column_if_present()`.  Added.